### PR TITLE
add TemplateFile to the templates so user can give a path to a template

### PR DESCRIPTION
**Pull request by xmcqueen**
seems useful. I'm doing it this way.

### CODE REVIEW

**Comment by sgreben** [comment]
Cool, thanks! LGTM.

**Comment by sgreben** [comment]
Ah right, could you also add a row to the overview table? https://github.com/sgreben/flagvar#types Then it's ready to merge.

**Comment by xmcqueen** [comment]
yes good idea

File: README.md [hunk 1]
```diff
@@ -122,6 +122,7 @@ Here's a compact overview:
 | [TCPAddrsCSV](https://godoc.org/github.com/sgreben/flagvar#TCPAddrsCSV)        | 127.0.0.1:10,:123               | []net.TCPAddr                              |
 | [Template](https://godoc.org/github.com/sgreben/flagvar#Template)    | "{{.Size}}"        | *template.Template                   |
 | [Templates](https://godoc.org/github.com/sgreben/flagvar#Templates)   | "{{.Size}}"        | []*template.Template                 |
+| [TemplateFile](https://godoc.org/github.com/sgreben/flagvar#TemplateFile)   | "{{.Size}}"        | String                 |
```
Comment (sgreben) [inline README.md]:
```suggestion
| [TemplateFile](https://godoc.org/github.com/sgreben/flagvar#TemplateFile)   | "/path/to/template.file"        | string                 |
```

Comment (xmcqueen) [reply README.md]: right. OK

**Comment by sgreben** [comment]
Merged. Thanks for the contribution @xmcqueen!

**Pull request merged**